src/main.rs

```diff
@@ -41,7 +41,20 @@ fn build_poi_properties(row: &Row) -> Result<Vec<Property>, String> {
             key: k,
             value: v.unwrap_or("".to_string()),
         })
-        .collect())
+        .collect::<Vec<Property>>();
+
+    let subclass = row.get_opt("subclass")
```

Comment:
I don't feel that subclass is a good name

Reply:
what about poi_subclass ?